### DISCUSSION TITLE
RuntimeError: submission must have at least 1 task

### DISCUSSION THREAD

**Discussion by kyladevilla** [deepmodeling/dpgen#640] [Q&A] [answered]
Hello, I am having the same issue as discussed in #547, where at the initial MD iteration I keep getting a "RuntimeError: submission must have at least 1 task."

However the solution posted there has not worked for me and I'd be very grateful for any help debugging the issue!

![image](https://user-images.githubusercontent.com/99236935/152922185-f6cfb327-9254-41c7-8004-889555760a30.png)

#547 indicated this was an issue with "sys_configs," but I believe I'm following the correct format: 
![image](https://user-images.githubusercontent.com/99236935/152922410-f03c1d56-60fc-49d7-bdef-62a47e2fca54.png)

I've also tried this with multiple lists in sys_configs, i.e. 
![image](https://user-images.githubusercontent.com/99236935/152922523-e09acfc0-61bf-412a-8a40-8b73488d07ef.png)

and without specifying a sys_configs_prefix but rather enumerating an entire path for the sys_configs: 
![image](https://user-images.githubusercontent.com/99236935/152922593-f315a6c7-ef35-4108-bc70-490166aa32a9.png)

and with using a wildcard character:
![image](https://user-images.githubusercontent.com/99236935/152922708-2952834d-f6bd-4363-94bd-952eed16bff1.png)

as well as every possible combination of leading and trailing slashes on both the "sys_configs_prefix" and "sys_batch_size" entries. 

I thought maybe my model_devi settings might be the issue but I believe these are also properly formatted:
![image](https://user-images.githubusercontent.com/99236935/152922838-a5119877-5f62-45ef-aa87-5aa35e7b4236.png)

and I have definitely confirmed this path exists: 
![image](https://user-images.githubusercontent.com/99236935/152923162-ab266702-3f96-4e07-a9b4-40e11ae55b52.png)

could anyone point me towards a solution? Thank you!

**Comment by AnguseZhang** [comment, marked as answer]
Let's assume the iteration index is 0. 
Then please first `ls iter.000000/01.model_devi/` and see whether there're some task folders named `task.xxx.xxxxxx`. 
If not, you can reset the last index in `record.dpgen` to be `0 2` and rerun `dpgen`.
Let's see what happens.

**Reply by kyladevilla** [reply]
Hi Anguse!

Thank you so much for the rapid reply.

I realized my problem was in part that I had initially had the "sys_configs" wrongly formatted such that the iter.000000/01.model_devi/confs/000.0000.poscar file was a link to root. After correcting the "sys_configs" format I hadn't deleted those bad files. Deleting the entire iter.000000/01.model_devi/ directory and rolling "record.dpgen" back to "0 2" resulted in the MD proceeding as expected. 

Thanks again!
